src/worker.js

```diff
@@ -1,9 +1,13 @@
+/* eslint-disable import/first */
+require('newrelic');
+
 import {} from 'dotenv/config';
 import throng from 'throng';
 import { logger, auditLogger } from './logger';
 import { scanQueue } from './services/scanQueue';
 import processFile from './workers/files';
 
+require('newrelic');
```

Comment:
The weird double require is showing up here, just make sure it goes away in the next merge of main.